xc7/tests/litex/minilitex/README.md

```diff
@@ -0,0 +1,5 @@
+# MiniLitex - Litex BaseSoC
```

Comment:
I think  `xc7/tests/soc/litex/mini` would probably be a better directory name?

Really the files under `xc7/tests/` should be numbered in the same way that the top level `tests` are numbered -- but that is something for a different pull request.

Reply:
Done, does look better this way. Opened an issue for the organization of the xc7 tests - https://github.com/SymbiFlow/symbiflow-arch-defs/issues/1194